derp/derp_test.go

```diff
@@ -60,7 +67,7 @@ func TestSendRecv(t *testing.T) {
 			t.Fatal(err)
 		}
 		defer cin.Close()
-		go s.Accept(cin, bufio.NewReadWriter(bufio.NewReader(cin), bufio.NewWriter(cin)))
+		go s.Accept(cin, bufio.NewReadWriter(bufio.NewReader(cin), bufio.NewWriter(cin)), "")
```

Comment:
use an obvious dummy value instead of an empty string literal?

Either a named const or a "dummy-ip:port" literal?